other-components.yml

```diff
@@ -5,16 +5,16 @@
 # 'services' tag.
 services:
     mender-artifact:
-        image: mendersoftware/mender-artifact:3.10.1
+        image: mendersoftware/mender-artifact:3.10.2
 
     mender-cli:
-        image: mendersoftware/mender-cli:1.11.0
+        image: mendersoftware/mender-cli:1.11.1
```

Comment:
I cant pull this image:
```bash
# docker pull mendersoftware/mender-cli:1.11.1
Error response from daemon: pull access denied for mendersoftware/mender-cli, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```
should it be like that?

Reply:
These are not Docker images (I know, I know).
It is just a way to express verisons.